static/app/views/issueList/groupSearchViewTabs/editableTabTitle.tsx

```diff
@@ -1,29 +1,38 @@
 import {useEffect, useRef, useState} from 'react';
+import {useTheme} from '@emotion/react';
 import styled from '@emotion/styled';
+import {FocusScope} from '@react-aria/focus';
 
 import {GrowingInput} from 'sentry/components/growingInput';
 
+interface EditableTabTitleProps {
+  isEditing: boolean;
```

Comment:
What does `isEditing` do now that the input is always being rendered? Seems like it should disable the input or something, but I don't see it doing much.

Reply:
Its a way for the parent component to control when the inputRef gets focused. I can try to refactor this so that the parent component creates the ref instead and focuses it when the rename button gets clicked.

Reply:
You could call `inputRef.current.focus()` on the double click event, right? Then you wouldn't need to do it in a useEffect

Reply:
Yup! but the parent component still needs some way to trigger the focus when the user clicks on the rename button from the tab's ellipsis menu

Reply:
Ohhhh I see now. Gotcha, keep it as is then!